src/components/BulkEditActionMenu/BulkEditActionMenu.js

```diff
@@ -100,7 +100,7 @@ const BulkEditActionMenu = ({
   const renderStartBulkEditButtons = () => {
     return (
       <>
-        {isStartBulkInAppActive && (
+        {isStartBulkInAppActive && !isCompleted && (
```

Comment:
you can add it [here](https://github.com/folio-org/ui-bulk-edit/pull/103/files#diff-ba4586c0afa30a5c1d713f14228b91194260a28b1da945dc7ca296567b468598R98)

Reply:
Thanks a lot! Already fixed it.